test/global_resource_test.cpp

```diff
@@ -28,18 +28,16 @@ std::size_t allocation_count = 0;
 #endif
 
 #if defined(BOOST_GCC) && (BOOST_GCC >= 40700) && (__cplusplus >= 201103L)
-#define BOOST_CONTAINER_NEW_EXCEPTION_SPECIFIER
```

Comment:
This triggers warnings in older platforms that Boost.Container still supports. I guess this should be detected depending on the compiler version and maybe added to Boost.Config.

Reply:
Current behavior is only for GCC, in commit:

https://github.com/boostorg/container/commit/485878d566c21d6029129769019ed96e0214b537

the exception specifier is not used if __cplusplus shows C++11.

Reply:
Great, thanks!

Now, due to your change, my PR#66 has stopped failing the CI tests on clang.